src/extensions/scratch3_boost/index.js

```diff
@@ -590,15 +594,16 @@ class Boost {
         this._sensors = {
             tiltX: 0,
             tiltY: 0,
-            color: BoostColor.NONE
+            color: BoostColor.NONE,
+            previousColor: BoostColor.NONE
         };
 
         /**
          * An array of values from the Boost Vision Sensor.
          * @type {Array}
          * @private
          */
-        this._colorBucket = [];
+        this._colorSamples = [];
```

Comment:
Does this change belong in another PR, or should the issue this resolves be mentioned in the comment for this PR?

Reply:
Yeah thanks for pointing that out @evhan55. It's from #2091!